src/System.CommandLine/Help/HelpOption.cs

```diff
@@ -9,7 +9,7 @@ internal class HelpOption : Option<bool>
         private string? _description;
 
         public HelpOption(string[] aliases, Func<LocalizationResources> getLocalizationResources)
-            : base(aliases)
+            : base(aliases, null, new Argument<bool> { Arity = ArgumentArity.Zero })
```

Comment:
Could `new Argument<bool> { Arity = ArgumentArity.Zero }` be in a static and be re-used here and on VerisonOption?

Reply:
IIRC I've tried it in the past (when I was working on SCL perf) and it was impossible as some of the methods were allowing for mutation, but I am not 100% sure if this is the case now.

Once we are done with API-redesign and re-layering we are going to profile the startup scenario again and if it pops up, try to cache it. But initializing static fields also has a cost, so this will have to be carefully verified.